src/expr/node_value.cpp

```diff
@@ -97,20 +97,20 @@ std::ostream& operator<<(std::ostream& out, const NodeValue& nv)
 
 void NodeValue::markRefCountMaxedOut()
 {
-  Assert(NodeManager::currentNM() != nullptr)
+  Assert(d_nm != nullptr)
```

Comment:
Nit: maybe we can move this assertion to the constructor?  This would make debug builds slightly faster?

Reply:
Unfortunately this is not possible. The `NodeValue` constructor does not initialize `d_nm`. See:
https://github.com/cvc5/cvc5/blob/main/src/expr/node_value.h#L288

The node manager pointer (and the rest of fields) are initialized in `NodeBuilder`.